helm/templates/deployment-postgres.yaml

```
@@ -15,6 +15,9 @@ spec:
     metadata:
       annotations:
         {{- include "checksumCm" (list $ "/cm-postgres.yaml") | nindent 8 }}
+        {{- range $key, $value := .Values.postgres.container.annotations }}
+        {{ $key }}: {{ $value | quote | nindent 8 }}
+        {{- end }}
```

```suggestion
       {{- .Values.postgres.container.annotations | toYaml | nindent 8 }}
```
Ceci devrait être équivalent sauf si le but était d'ajouter les `quote` qui seraient absents du values